policyengine_us/parameters/gov/states/ne/tax/income/credits/ctc/refundable/fpg_fraction.yaml

```
@@ -0,0 +1,14 @@
+description: Nebraska limits the refundable child tax credit to filers with an adjusted gross income amount at or below this fraction of federal poverty guidelines.
```

To reflect the law. Note in a comment in the variable that they define total household income as federal AGI, per the DOR site.
```suggestion
description: Nebraska limits the refundable child tax credit to filers with an total household income at or below this fraction of the federal poverty guideline.
```